### ISSUE TITLE
Links should be identified correctly

### ISSUE THREAD

**Issue by HRemonen** [UniversityOfHelsinkiCS/palaute#1241]
In some places links are styled as buttons or chips which breaks success criterion 1.4.1.
Links must be identified by underlines or some other non-color visual distinction.

While some links may be visually evident from page design and context, such as navigational links, links within text are often visually understood only from their own display attributes. Removing the underline and leaving only the color difference for such links would be a failure because there would be no other visual indication (besides color) that it is a link.

Also links must be presented with an underline on hover. See also requirements for adequate color contrast ratios:
https://webaim.org/resources/linkcontrastchecker/

**Comment by Veikkosuhonen** [comment]
Good one

**Comment by HRemonen** [comment]
We'll see when the time comes for this one tho :D There's a lot of pages and links to go through

**Comment by Veikkosuhonen** [comment]
Start from https://github.com/UniversityOfHelsinkiCS/palaute/blob/master/src/client/components/common/LinkButton.js. That should fix most of the issues.
